src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1039,9 +1039,9 @@ private bool CheckFieldValueKind(SyntaxNode node, BoundFieldAccess fieldAccess,
                         fieldIsStatic == containing.IsStatic &&
                         (fieldIsStatic || fieldAccess.ReceiverOpt.Kind == BoundKind.ThisReference) &&
                         (Compilation.FeatureStrictEnabled
-                            ? TypeSymbol.Equals(fieldSymbol.ContainingType, containing.ContainingType, TypeCompareKind.ConsiderEverything2)
+                            ? TypeSymbol.Equals(fieldSymbol.ContainingType, containing.ContainingType, TypeCompareKind.AllNullableIgnoreOptions)
```

Comment:
>AllNullableIgnoreOptions [](http://example.com/codeflow?start=119&length=24)

I think we should use ```AllIgnoreOptions``` here.

Reply:
Does it matter for the purpose of this specific check?

Reply:
> Does it matter for the purpose of this specific check?

The question is what else ignorable do we want to take into account and why. We want to make a change that is good enough for the future as well. If we remove ```ConsiderEverything2```, it will be unlikely that we will focus on this line any time soon. Imagine we will add another feature similar to nullable. Is this line going to break again? We should pick an option that matches what we are trying to accomplish as close as possible, not one that merely makes a test to pass.

Reply:
@AlekseyTs Got it. I asked in case there is already a scenario I should test.